script/bootstrap

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# script/bootstrap: Resolve all dependencies that the application requires to
+#                   run.
```

Comment:
Why the massive empty space before `run.`?

Reply:
It's literally just been lifted from https://github.com/github/scripts-to-rule-them-all/blob/master/script/bootstrap. I think it's so the `script/bootstrap` looks more like an item in a list, but I'm absolutely not precious about it.

Reply:
The really short line must come from some really strict adherence to "wrap the line as if people are reading code on a phone" rule... Oh well, not a hill I want to sustain even minor injuries on right now.